### PR TITLE
Support signing deposits, withdrawals and orders with Bitcoin wallet in UI

### PR DESCRIPTION
**Pull request by antonyshchenko**
Wallet mod: https://github.com/funkybit-labs/xverse-wallet-mod

Demo:

https://github.com/user-attachments/assets/254b3aff-97e2-4891-b3e7-b0b3b928dc99

### CODE REVIEW

File: backend/src/main/kotlin/xyz/funkybit/core/model/TxHash.kt [hunk 1]
```diff
@@ -22,4 +22,7 @@ value class TxHash(val value: String) {
 
         fun fromDbModel(txHash: xyz.funkybit.core.model.db.TxHash): TxHash = TxHash(txHash.value)
     }
+
+    // TODO: get rid of xyz.funkybit.core.model.db.TxHash class
```
Comment (bholzman) [inline backend/src/main/kotlin/xyz/funkybit/core/model/TxHash.kt:26]: add a ticket to backlog to do this?

File: web-ui/src/components/Screens/HomeScreen/DepositModal.tsx [hunk 2]
```diff
@@ -88,54 +101,67 @@ export default function DepositModal({
...
+              to: depositAddress as Address,
+              value: amount
+            })
+          }
         } else {
-          setSubmitPhase('waitingForDepositApproval')
-          depositHash = await sendTransaction(config, {
-            to: exchangeContractAddress as Address,
-            value: amount
+          const response = await Wallet.request('sendTransfer', {
```
Comment (bholzman) [inline web-ui/src/components/Screens/HomeScreen/DepositModal.tsx:152]: should set a submit phase here so we can update the modal button text accordingly, no?

File: web-ui/src/components/Screens/HomeScreen/WithdrawalModal.tsx [hunk 3]
```diff
@@ -174,7 +159,7 @@ export default function WithdrawalModal({
                 </div>
 
                 <p className="mb-2 mt-4 text-center text-sm text-darkBluishGray2">
-                  Wallet balance:{' '}
+                  Available balance:{' '}
```
Comment (bholzman) [inline web-ui/src/components/Screens/HomeScreen/WithdrawalModal.tsx:162]: 👍

File: web-ui/src/components/Screens/HomeScreen/DepositModal.tsx (hunk 2 above)
Comment (antonyshchenko) [reply web-ui/src/components/Screens/HomeScreen/DepositModal.tsx:153]: yup, thanks

File: backend/src/main/kotlin/xyz/funkybit/core/model/TxHash.kt (hunk 1 above)
Comment (antonyshchenko) [reply backend/src/main/kotlin/xyz/funkybit/core/model/TxHash.kt:29]: yup, done

**Pull request merged**